trulens_eval/examples/updated_dashboard_pt.2.ipynb

```diff
@@ -0,0 +1,13086 @@
+{
```

Comment:
<p>Can you streamline this setup? The key point to get across to the reader is that they need to bring their own record metadata (the title here) and then your change finds hotspots for them.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1160/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>